src/components/FAQ.vue

```diff
@@ -289,6 +289,9 @@ export default {
     // justify-content: center;
     flex-wrap: nowrap;
     height: auto;
+    width: calc(100% + 2rem);
+    margin-left: -1rem;
+    font-size: 0.85rem;
```

Comment:
This is boy-scouting this bug I discovered where this menu looks cropped in mobile-size width:

[ Before ]

<img src='https://github.com/user-attachments/assets/09b6c100-2ef4-46eb-a53c-307835ef3107' width='420'>

[ After ]

<img src='https://github.com/user-attachments/assets/55223842-b91e-46fa-81d9-4f78f0f5a0c2' width='420'>